validation/rbac/psa/deprecated_restrictedadmin_test.go

```diff
@@ -0,0 +1,249 @@
+//go:build (validation || infra.any || cluster.any || stress) && !sanity && !extended && (2.8 || 2.9 || 2.10)
```

Comment:
If we don't want to run this on 2.7, it should be `!2.7` to avoid changing it when Rancher versions change.